pkg/snowflake/tag_association.go

```diff
@@ -21,19 +21,7 @@ type TagAssociationBuilder struct {
 }
 
 type tagAssociation struct {
-	ColumnId       sql.NullString `db:"COLUMN_ID"`
-	ColumnName     sql.NullString `db:"COLUMN_NAME"`
-	Domain         sql.NullString `db:"DOMAIN"`
-	ObjectDatabase sql.NullString `db:"OBJECT_DATABASE"`
-	ObjectDeleted  sql.NullString `db:"OBJECT_DELETED"`
-	ObjectId       sql.NullString `db:"OBJECT_ID"`
-	ObjectName     sql.NullString `db:"OBJECT_NAME"`
-	ObjectSchema   sql.NullString `db:"OBJECT_SCHEMA"`
-	TagDatabase    sql.NullString `db:"TAG_DATABASE"`
-	TagId          sql.NullString `db:"TAG_ID"`
-	TagName        sql.NullString `db:"TAG_NAME"`
-	TagSchema      sql.NullString `db:"TAG_SCHEMA"`
-	TagValue       sql.NullString `db:"TAG_VALUE"`
+	TagValue sql.NullString `db:"TAG_VALUE"`
```

Comment:
It seems like `tagAssociation` is never accessed, so removing the attributes shouldn't harm anything.

`ListTagAssociations` is only called twice in `pkg/resources/tag_association.go`, one time the result isn't used at all, and one time only the object count of the array is accessed.